model/ontology/taotest.rdf

```diff
@@ -18,4 +18,13 @@
     <tao:TAOGUIOrder><![CDATA[1]]></tao:TAOGUIOrder>
   </rdf:Description>
 
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAOTest.rdf#AssessmentProjectId">
```

Comment:
We already have this property for delivery form `http://www.tao.lu/Ontologies/TAODelivery.rdf#AssessmentProjectId`. It will be redundant and confuse. 

This extension already depends on delivery, so we do not need to add it here again. 

Suggestion: This new property `http://www.tao.lu/Ontologies/TAOTest.rdf#AssessmentProjectId` can be added only inside the Test extension and while receiving the publishing the test you can migrate this property to the corresponding one `http://www.tao.lu/Ontologies/TAODelivery.rdf#AssessmentProjectId`

Reply:
Hey @gabrielfs7, this is exactly the plan. When the feature is totally implemented, the Property `http://www.tao.lu/Ontologies/TAODelivery.rdf#AssessmentProjectId.` will be removed from the delivery. 
You can check the full plan here for more details:
https://oat-sa.atlassian.net/wiki/spaces/NEX/pages/1674117121/Remote+Publishing+-+Publish+without+generating+local+delivery

Reply:
So you mean that the delivery wont have the `AssessmentProjectId` anymore at all? It will remain only in the test? I could not find this in the plan.

Reply:
No, sorry, my mistake it will have in both places as we want to publish the tests in the test level and on the delivery level.